### PR TITLE
added CountryCurrency

### PR DESCRIPTION
**Pull request by rm-Umar**

### CODE REVIEW

**Review by UmanShahzad — APPROVED** [review]
@SobanMahmood please also do check.

**Comment by rm-Umar** [comment]
I've added `Continent` in this PR as well. 
cc: @UmanShahzad @SobanMahmood

File: src/IPinfo/Models/Continent.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using System.Text.Json.Serialization;
+namespace IPinfo.Models
+{
+  /// <summary>
+  /// Gets continent code and name.
+  /// { "code": "PKR" ,"name": "Asia"}
```
Comment (SobanMahmood) [inline src/IPinfo/Models/Continent.cs]:
```suggestion
  /// { "code": "AS", "name": "Asia"}
```

File: README.md [hunk 2]
```diff
@@ -106,6 +107,18 @@ Console.WriteLine($"IPResponse.CountryFlag.Emoji: {ipResponse.CountryFlag.Emoji}
...
+// currency code, e.g. "US" -> "USD"
+Console.WriteLine($"IPResponse.CountryCurrency.Code: {ipResponse.CountryCurrency.Code}");
+
+// currency symbol, e.g. "US" -> "$"
+Console.WriteLine($"IPResponse.CountryCurrency.Symbol: {ipResponse.CountryCurrency.Symbol}");
+
+// continent code, e.g. "US" -> "NA"
+Console.WriteLine($"IPResponse.Continent.Code: {ipResponse.Continent.Code}");
+
+// continent name , e.g. "US" -> "North America"
```
Comment (SobanMahmood) [inline README.md]:
```suggestion
// continent name, e.g. "US" -> "North America"
```

**Review by SobanMahmood — CHANGES_REQUESTED** [review]
Looks good to me, just minor updates in comments/readme.

**Comment by rm-Umar** [comment]
If everything seems good we can merge it.
@UmanShahzad @SobanMahmood

**Review by SobanMahmood — APPROVED** [review]

**Pull request merged**